section2/reflection.md

```diff
@@ -2,28 +2,113 @@
 
 1. Regarding the blog posts in Part A, how do you feel about asking questions? Do you tend to ask them too soon, or wait too long, or somewhere in between?
 
+I love asking questions. I've found it to be a necessity in my current and previous employment. In my current employment, I work with engineers who will spend hours on a formatting issue in MS Office, which is something I could fix in less than 5 minutes. It's maddening when they do that and such a huge waste of their time. In the "Your Questions are Dumb" article, the author mentions this kind of activity. It hit very close to home!
+
+I don't have very much ego when it comes to asking questions, and I can completely acknowledge that I am not the smartest person in the room! I do like finding my own solutions, and that is always my go-to way to do things first. But I will definitely use the 20-minute timer for searches and reach out for help when needed.
+
+The article of these three that resonated with me the most was the "Asking Better Questions" article. Formulating search strategies is something I find personally fascinating. I think there is something so satisfying about developing a strategy and successfully executing it (esp. if it's the first thing you try!).
+
+I really like the examples of good Slack questions on the Turing Questions page. That is very helpful.
+
 ### If Statements
 
 1. What is a conditional statement? Give three examples.
 
+Conditional statements create forks in the road. We often use if, elsif, and else to determine which fork to go down next.
+Ex1: weight = 12
+if weight < 10
+ puts "That's a petit steak."
+elsif weight > 10
+ puts "That's a whole cow!"
+else
+ puts "perfect size for fancy Friday dinner."
+end
+
+Ex2: temp = 95
+if temp > 95
+ puts "Hot!! More like the surface of the sun than a season!"
+elsif temp < 95 || temp > 75
+ puts "Summer weather"
+else
+ puts "It's not summer."
```

Comment:
I think this `else` would also be returned if the temp was exactly `95`, so try to think of those edge cases when you are writing your code and maybe run it in the terminal with a few different values to check your work.

Reply:
Also, looking at line 30 I think what you were trying to say that a temp between 95 and 75 is "Summer Weather"? In that case you, just like when you express it in English, you'd want to use `and` instead of `or`. Here I believe a temp value of `1` would return `summer weather`. Try it out in the terminal!

Reply:
I re-wrote this as follows and was able to get the expected outputs:

if temp > 95
 puts "Hot!! More like the surface of the sun than a season!"
elsif temp <= 95 && temp > 75
 puts "Summer weather"
else
 puts "It's not summer."
end